app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -135,6 +134,11 @@ public static DocumentEditFragment newInstance(File path, boolean pathIsFolder,
     private MarkorWebViewClient _webViewClient;
     private boolean _nextConvertToPrintMode = false;
     private long _loadModTime = 0;
+    private boolean _isTextChanged = true; // true forces first update
+    private Activity _activity;
```

Comment:
As much as I hate calling `getActivity()` instead of simply accessing it directly via some var .. I think it's better than storing it as hard member reference. On device rotation there might be also a point where the member reference is outdated, but `getAcitvity()` would already return the right thing. (And activity leak)

Reply:
I mean, the same logic applies to _webView, _shareUtil, _appSettings etc right? We are assigning `_activity` in `onViewCreated`, which should properly happen _after_ rotation etc.

I can revert it. But if we do, imo we should think about what else to revert to direct access

Reply:
this is correct, but the utilities don't have a way to access activity otherwise than passing it somewhere. at fragments theres the builtin `getActivity` which always returns the right thing 

do you see an issue with it like getActivity being much slower than storing and using member?

Reply:
> do you see an issue with it like getActivity being much slower than storing and using member?

There is no performance impact, only asthethic. getActivity just returns a member from the base class.

I'll just revert it - reduces the complexity in this PR.